core/src/main/java/nu/yona/server/subscriptions/service/BuddyDisconnectMessageDto.java

```diff
@@ -138,7 +136,7 @@ private MessageActionDto handleAction_Process(UserDto actingUser, BuddyDisconnec
 			return MessageActionDto.createInstanceActionDone(theDtoFactory.createInstance(actingUser, messageEntity));
 		}
 
-		private void logHandledAction_Process(UserDto actingUser, BuddyDisconnectMessage messageEntity)
+		private void logHandledAction_Process(User actingUser, BuddyDisconnectMessage messageEntity)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://10.127.10.107:9000/coding_rules#rule_key=squid%3AS00100)